pages/measurement/[[...report_id]].js

```diff
@@ -40,7 +40,7 @@ export async function getServerSideProps({ query }) {
   // in which case, the extra segments are available inside query.report_id[1+]
   const report_id = query?.report_id?.[0]
   // If there is no report_id to use, fail early with MeasurementNotFound
-  if (typeof report_id !== 'string' || report_id.length < 1) {
+  if (typeof report_id !== 'string' || !report_id.match(/[a-zA-Z0-9_-]{5,100}/)) {
```

Comment:
I think we can be even a bit more aggressive with the filter here by setting the minimum length to 40 (according to the fastpath table the shortest non-null report_id is 45 characters, and it's of the new format `20201028T073929Z_ndt_ZZ_0_n1_Ed95brGpL3sZ4dUN`, so even in the case of a single character test name we would be at 43 chars).